sunpy/map/mapbase.py

```diff
@@ -672,10 +680,12 @@ def unit(self):
 
     def _base_name(self):
         """Abstract the shared bit between name and latex_name"""
-        return "{nickname} {{measurement}} {date}".format(
-            nickname=self.nickname,
-            date=parse_time(self.date).strftime(TIME_FORMAT)
-        )
+        if self.measurement is None:
+            format_str = "{nickname} {date}"
+        else:
+            format_str = "{nickname} {{measurement}} {date}"
+        return format_str.format(nickname=self.nickname,
+                                 date=parse_time(self.date).strftime(TIME_FORMAT))
```

Comment:
How about this?
```suggestion
        segments = [self.nickname, self.date.strftime(TIME_FORMAT)]
        if self.measurement is not None:
            segments.insert(1, "{measurement}")
        return " ".join(segments)
```

This also drops the `parse_time` as that should have been called in `self.date` *riiight*?